api/v2/checluster_types.go

```diff
@@ -90,6 +90,16 @@ type CheClusterDevEnvironments struct {
 	// These default components are meant to be used when a Devfile, that does not contain any components.
 	// +optional
 	DefaultComponents []devfile.Component `json:"defaultComponents,omitempty"`
+	// Idle timeout for workspaces in seconds.
+	// This timeout is the duration after which a workspace will be idled if there is no activity.
+	// To disable workspace idling due to inactivity, set this value to -1.
+	// +kubebuilder:default:=900
+	SecondsOfInactivityBeforeIdling *int32 `json:"secondsOfInactivityBeforeIdling,omitempty"`
+	// Run timeout for workspaces in seconds.
+	// This timeout is the maximum duration a workspace runs.
+	// To disable workspace run timeout, set this value to -1.
+	// +kubebuilder:default:=-1
```

Comment:
Pls. mention the defaults here as well
https://github.com/eclipse-che/che-operator/blob/885d623258450b6c942d7396c4f3c17ff05b0f2e/api/v2/checluster_types.go#L40